src/app/app.component.html

```diff
@@ -206,6 +206,7 @@ <h4 class="mb-4">2019/20 SKYSTONE Kickoff</h4>
             <mdc-list-item><mdc-radio mdcListItemGraphic value="en"></mdc-radio>🇺🇸 English</mdc-list-item>
             <mdc-list-item><mdc-radio mdcListItemGraphic value="he"></mdc-radio>🇮🇱 Hebrew</mdc-list-item>
             <mdc-list-item><mdc-radio mdcListItemGraphic value="es"></mdc-radio>🇪🇸 Spanish</mdc-list-item>
+            <mdc-list-item><mdc-radio mdcListItemGraphic value="zh"></mdc-radio>zh Chinese(Simplified)</mdc-list-item>
```

Comment:
Please use ``🇨🇳`` (copy and paste) instead of the ``zh``.
(It's the china flag, but [flag emojis are missing from Windows](https://answers.microsoft.com/en-us/windows/forum/all/flag-emojis-missing-from-windows-10/14b124f9-7382-4a6d-868c-52ee3cb977be) so you might see it as text)